common/planners/src/plan_expression.rs

```diff
@@ -51,6 +55,8 @@ pub enum Expression {
     Literal(DataValue),
     /// select * from t where xxx and exists (subquery)
     Exists(Arc<PlanNode>),
+    /// select number from t where number in (1, 3, 5)
+    InList(InListExpr),
```

Comment:
Maybe it's better to use Expression::Literal?

Reply:
For `field_a IN (1, 2, 3, 4, 5, 6)` it will be constructed as:
```
Expression::ScalarFunction {
    op : "in",
    args: [
        Expression::Column("field_a"),
        Expression::Literal(DataValue::List(vec![DataValue::UInt8(1), ..., DataValue::UInt8(6)]))
    ]
}
```

Reply:
ScalarFunction does not have the `negated` field for InList.

https://docs.rs/sqlparser/0.9.0/sqlparser/ast/enum.Expr.html#variant.InList